resources/_gen/assets/css/ananke/css/main.css_bb5467e0521bbea6b1e66429f6ec028e.json

```diff
@@ -0,0 +1 @@
+{"Target":"ananke/css/main.min.css","MediaType":"text/css","Data":{}}
```

Comment:
Are you sure this should be versioned? 🤔 Judging by the name, I'm under the impression that `resources/_gen` should be ignored (maybe even `resources`).

Reply:
I did the change to ignore it 👍

Reply:
But you still need to `git rm` these files now 😉 Once they are added, they will be tracked even if ignored. Ideally, make that a `fixup!` of the commit when they were added so we don't have an add / remove two commits apart.